chipdrivers.h

```diff
@@ -24,10 +24,12 @@
 
 /* spi.c */
 int spi_aai_write(struct flashctx *flash, const uint8_t *buf, unsigned int start, unsigned int len);
+int default_spi_write_256(struct flashctx *flash, const uint8_t *buf, unsigned int start, unsigned int len); // it uses spi_write_chunked
```

Comment:
This doesn't seem like it belongs here (it's already in programmer.h)

Reply:
I need a function for this chip that can handle 32 bytes page write. What is the problem with putting this declaration here and what would be a better way?

Reply:
ok, spi_chip_write_256 can handle smaller pages eg. 32 bytes